CONTRIBUTING.md

```diff
@@ -43,6 +43,8 @@ For reference, here are descriptions of what the other 'TODO' tags mean:
 
 ## Instructions for making a code change
 
+**Working on your first Pull Request?** You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github)
```

Comment:
Add colon after "series"; add full stop at end. I think it's also fine to de-italicize 'free' -- we're starting to overuse italics a bit in this section to convey different things.